### ISSUE TITLE
Still getting the conflicts

### ISSUE THREAD

**Issue by iamgojoof6eyes** [bisohns/search-engine-parser#178]
[Context](https://github.com/bisohns/search-engine-parser/issues/175)

Even after upgrading it is still creating conflict while deploying the app
[![Pic](https://te.legra.ph/file/d42f7d384d6d8193e4d01.jpg)]

**Comment by WolfgangFahl** [comment]
```bash
The conflict is caused by:
    pysotsog 0.0.13 depends on lxml>=4.9.1
    search-engine-parser 0.6.7 depends on lxml==4.6.5
```
Why the ==?

**Comment by iamgojoof6eyes** [comment]
> ```shell
> The conflict is caused by:
>     pysotsog 0.0.13 depends on lxml>=4.9.1
>     search-engine-parser 0.6.7 depends on lxml==4.6.5
> ```
> 
> Why the ==?

Not a compulsion to use ==, do you mean if I use >= it should resolve the problem right?

**Comment by MeNsaaH** [comment]
Fixed in #181
